sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/utils/AbstractRuleExampleTest.java

```diff
@@ -112,6 +113,53 @@ private String displayName(Rule rule, RuleExample example) {
     return String.format("%s example %d: Should%s find issue", rule.getId(), indexOfExampleInRule, positiveOrNegative);
   }
 
+  private String failingMessage(Collection<Issue> issues, Rule rule, RuleExample ruleExample) {
+    StringBuilder sb = new StringBuilder();
+    sb.append("Test case \"");
+    sb.append(displayName(rule, ruleExample));
+    if (issues.isEmpty()) {
+      sb.append("\" failed as it didn't match on any content");
+      return sb.toString();
+    }
+
+    sb.append("\" failed as it matched on the following content:\n");
```

Comment:
Should we care about using `System.lineSeparator()` here instead of `\n`? Or do IDEs display it correctly also on Windows?

Reply:
Good catch, I will change it. It works on windows, I'm not sure about mac